javascripts/discourse/components/discourse-tag-banners.hbs

```diff
@@ -0,0 +1,21 @@
+{{#if this.isVisible}}
+  {{#if this.loaded}}
+    <div
+      class="tag-title-header tag-banner-{{this.formattedTagName}}
+        {{this.additionalClass}}"
+    >
```

Comment:
You can use `class={{concat-class "tag-title-header" (concat "tag-banner-" this.formattedTagName) this.additionalClass}}`